### ISSUE TITLE
crash on save

### ISSUE THREAD

**Issue by cosmoderp** [EDDiscovery/EDDiscovery#1793]
the program no longer remembers pop-out positions as before.  
So I tried 5 times to do it manually.  When I click "Save Current Setup"  it crashes and 
windows throws this:

```
There was an unhandled UI exception.

==== BEGIN ====
System.NullReferenceException: Object reference not set to an instance of an object.
   at EDDiscovery.UserControls.UserControlMap.textMaxRadius_ValueChanged(Object sender, EventArgs e) in C:\Code\EDDiscovery\EDDiscovery\UserControls\UserControlMap.cs:line 236
   at ExtendedControls.NumberBox`1.Timer_Tick(Object sender, EventArgs e) in C:\Code\EDDiscovery\ExtendedControls\Controls\NumberBox.cs:line 148
   at System.Windows.Forms.Timer.OnTick(EventArgs e)
   at System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
===== END =====
```

**Comment by robbyxp1** [comment]
Investigating

**Comment by robbyxp1** [comment]
Found will be fixed. thanks for reporting

**Comment by robbyxp1** [comment]
Fixed in 9.1.6 to come today
